### ISSUE TITLE
[zeroname] error updater

### ISSUE THREAD

**Issue by rllola** [HelloZeroNet/ZeroNet#1895]
I am trying to run the zeroname plugin with namecoin 16.3

I have the following error when trying to run `zeroname_updater.py`:
```
bitcoin-cli has the option -getinfo to collect and format these in the old format.
Exception <class 'bitcoinrpc.authproxy.JSONRPCException'> -32601: getinfo

This call was removed in version 0.16.0. Use the appropriate fields from:
- getblockchaininfo: blocks, difficulty, chain
- getnetworkinfo: version, protocolversion, timeoffset, connections, proxy, relayfee, warnings
- getwalletinfo: balance, keypoololdest, keypoolsize, paytxfee, unlocked_until, walletversion
```

Maybe we could add a line to make it compatible ?

**Comment by rllola** [comment]
I can confirm than modifying this line : 
https://github.com/HelloZeroNet/ZeroNet/blob/master/plugins/Zeroname/updater/zeroname_updater.py#L175

to use `getblockchaininfo` it works.

**Comment by rllola** [comment]
This PR should fix it https://github.com/HelloZeroNet/ZeroNet/pull/1896

**Comment by rllola** [comment]
Fixed with #1896.
